### PR TITLE
docs: Update CVO doc for run level and parallelization

### PR DESCRIPTION
**Pull request by smarterclayton**
Also document that ordering depends on proper ClusterOperator implementation
and components should transition to utilizing direct dependencies.

### CODE REVIEW

**Comment by smarterclayton** [comment]
/assign @abhinavdahiya 

Document the behavior of 4.0 CVO

File: docs/dev/operators.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+`/release-manifests` within that image. The `/manifests` directory is part of the CVO image and
+contains the basic deployment and other roles for the CVO. The `/release-manifests` directory is
+created by the `oc adm release new` command from the `/manifests` directories of all other
+candidate operators.
+
+The contents of `/release-manifests` are applied in lexigraphic order, exactly as `ls` would
+return on a standard Linux or Unix derivative. The CVO supports the idea of "run levels" by
+defining a convention for how operators that wish to run before other operators should name
+their manifests. A run level is of the form `0000_\d\d_[a-z0-9\-]+_<filename>` where the first
+digits are the level (see below for assignment), the second chunk is the component name that
```
Comment (abhinavdahiya) [inline docs/dev/operators.md]: `(see below for assignment)` maybe link to heading?

**Comment by smarterclayton** [comment]
integration is flaking on a similar change to what I fixed in the other test

**Comment by smarterclayton** [comment]
Doc fixed, any other comment?

**Pull request merged**